Dockerfile.old-test-suite

```diff
@@ -4,10 +4,8 @@ RUN /root/bin/docker-bootstrap.sh memcached
 RUN /root/bin/docker-bootstrap.sh backend
 RUN /root/bin/docker-bootstrap.sh frontend
 
-ADD src/api/Gemfile /obs/src/api/Gemfile
-ADD src/api/Gemfile.lock /obs/src/api/Gemfile.lock
+ADD --chown=frontend:users src/api/Gemfile* /obs/src/api/
```

Comment:
Nitpick. Same as previous comment.

Is `:users` needed? I would leave this line as:

`ADD --chown=frontend src/api/Gemfile* /obs/src/api/`